### DISCUSSION TITLE
Google colab

### DISCUSSION THREAD

**Discussion by psythemexx** [lllyasviel/Fooocus#2848] [Q&A]
How to Run it on google colab ?

**Comment by IPv6** [comment]
Free tier seems to be blocked soon - already happens for some user, since there is a lot of new ones lately. As happened with WebUI back in the day //

On commercial tier just open [fooocus_colab.ipynb](https://github.com/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb) in collab

**Reply by ILikeToasters** [reply]
Are you using with the paid tier? I am curious if the public link issue is just for people using the free tier. I use locally 99% of the time myself but I have had several people ask about if this issue is only happening on the free tier.

**Reply by mashb1t** [reply]
public Gradio Share API links have nothing to do with Google Colab and they're not blocking anybody. This is purely an issue on Gradio's side.

Also see https://github.com/lllyasviel/Fooocus/issues/2852#issuecomment-2091534293
